mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -626,36 +635,45 @@ ::mediapipe::Status GlAnimationOverlayCalculator::GlSetup() {
 
     // vertex position in threespace
     attribute vec4 position;
+    attribute vec3 normal;
 
     // texture coordinate for each vertex in normalized texture space (0..1)
     attribute mediump vec4 texture_coordinate;
 
     // texture coordinate for fragment shader (will be interpolated)
-    varying mediump vec2 sample_coordinate;
+    varying mediump vec3 sample_coordinate;
+    varying mediump vec3 vNormal;
 
     void main() {
-      sample_coordinate = texture_coordinate.xy;
+      sample_coordinate = texture_coordinate.xyz;
       mat4 mvpMatrix = perspectiveMatrix * modelMatrix;
       gl_Position = mvpMatrix * position;
+
+      vec4 tmpNormal = mvpMatrix * vec4(normal, 1.0);
+      vNormal = normalize(tmpNormal.xyz);
     }
   )";
 
   const GLchar *frag_src = R"(
     precision mediump float;
 
-    varying vec2 sample_coordinate;  // texture coordinate (0..1)
+    varying vec3 sample_coordinate;  // texture coordinate (0..1)
+    varying vec3 vNormal;
     uniform sampler2D texture;  // texture to shade with
 
+    // Define ambient lighting constants
+    const vec3 ambient = vec3(-0.1, -0.1, -0.1);
```

Comment:
Having a negative ambient value doesn't make sense.  In order to "dim" or "brighten" we should be using multiplication in any event.  So for now you can try something like "vec3 ambient = (0.6, 0.6, 0.6)", and then have "gl_FragColor = vec4(pixelRGB * ambient, 1.0)".  If ambient is always going to be white/gray/untinted light, then we could even just make it a float constant.